cdc/api/v2/model.go

```diff
@@ -13,8 +13,36 @@
 
 package v2
 
+import (
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/pkg/config"
+	"github.com/pingcap/tiflow/pkg/security"
+)
+
 // Tso contains timestamp get from PD
 type Tso struct {
 	Timestamp int64 `json:"timestamp"`
 	LogicTime int64 `json:"logic-time"`
 }
+
+// Tables contains IneligibleTables and EligibleTables
+type Tables struct {
+	IneligibleTables []model.TableName `json:"ineligible-tables,omitempty"`
+	EligibleTables   []model.TableName `json:"eligible-tables,omitempty"`
+}
+
+// VerifyTableConfig use to verify tables.
+// Only use by Open API v2.
+type VerifyTableConfig struct {
+	PDAddrs       []string              `json:"pd-addrs"`
+	Credential    *security.Credential  `json:"credential"`
+	ReplicaConfig *config.ReplicaConfig `json:"replica-config"`
```

Comment:
we can define a new ReplicaConfig struct in this model file, otherwise if we change the ReplicaConfig , the open api behavior will be impacted